posthog/models/filters/retention_filter.py

```diff
@@ -40,7 +40,6 @@ class RetentionFilter(
     PropertyMixin,
     DisplayDerivedMixin,
     InsightMixin,
-    IntervalMixin,
```

Comment:
@EDsCODE Also removed IntervalMixin which I think is correct, as we use period here

Reply:
yup! I also removed some extraneous code i noticed